src/routes/index.ts

```diff
@@ -144,6 +144,8 @@ router.post(config.PRESENTER_WITH_PARAMS_URL, authentication, navigation.hasOver
 
 router.get(config.WHO_IS_MAKING_FILING_URL, authentication, navigation.hasPresenter, whoIsMakingFiling.get);
 router.post(config.WHO_IS_MAKING_FILING_URL, authentication, navigation.hasPresenter, ...validator.whoIsMakingFiling, checkValidations, whoIsMakingFiling.post);
+router.get(config.WHO_IS_MAKING_FILING_WITH_PARAMS_URL, authentication, navigation.hasPresenter, whoIsMakingFiling.get);
```

Comment:
Could you please move this line up, below 145. So we keep the 'gets' together and the 'posts' together. Consistent with what was done for presenter.

Reply:
Moved.